link/link_test.go

```diff
@@ -52,6 +54,49 @@ func TestRawLink(t *testing.T) {
 
 func TestRawLinkLoadPinnedWithOptions(t *testing.T) {
 	cgroup, prog := mustCgroupFixtures(t)
+	link, path := newRawLink(t, cgroup, prog)
+	c := qt.New(t)
```

Comment:
Can you change tests to use `qt.Assert` directly? `qt.New` might go away in the future.